Doc/c-api/datetime.rst

```diff
@@ -75,6 +75,18 @@ Type-check macros:
    *NULL*.
 
 
+.. c:function:: int PyTimeZone_Check(PyObject *ob)
+
+   Return true if *ob* is of type :c:data:`PyDateTime_TimeZoneType` or a subtype of
+   :c:data:`PyDateTime_TimeZoneType`.  *ob* must not be *NULL*.
+
+
+.. c:function:: int PyTimeZone_CheckExact(PyObject *ob)
```

Comment:
Since we are not exposing  PyDateTime_TimeZone struct, I don't see the utility of having PyTimeZone_Check macros.  Maybe something like PyTZInfo_Check as an accelerated version of `isinstance(x, datetime.tzinfo)` will be useful once we expose C versions of `tzinfo.utcoffset`, etc. methods, but this is better left for a separate PR.

Reply:
Why not just remove the `_Check` and leave the `_CheckExact`? I would think that the `_CheckExact` function is still meaningful.